From 23c9599c823ecf4fcb443442c5dbd95b885f2813 Mon Sep 17 00:00:00 2001
From: Fang Hui <b31070@freescale.com>
Date: Tue, 19 Aug 2014 14:52:14 +0800
Subject: [PATCH 522/635] ENGR00327640 - [sabresd_6sx]: cts testCameraFeatures
 failed

HAL set FOCUS_MODE_AUTO, but not exported in system feature file
/etc/permission/required_hardware.xml

Set FOCUS_MODE_FIXED can fix it.

Can't just remove, since if no focus mode, exception in
android.hardware.Camera$Parameters.set.

Signed-off-by: Fang Hui <b31070@freescale.com>
---
 mx6/libcamera/CameraBridge.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/mx6/libcamera/CameraBridge.cpp b/mx6/libcamera/CameraBridge.cpp
index 196c8d5..76232d1 100644
--- a/mx6/libcamera/CameraBridge.cpp
+++ b/mx6/libcamera/CameraBridge.cpp
@@ -233,9 +233,9 @@ status_t CameraBridge::initParameters(CameraParameters& params)
                CameraParameters::SCENE_MODE_AUTO);
 
     params.set(CameraParameters::KEY_SUPPORTED_FOCUS_MODES,
-               CameraParameters::FOCUS_MODE_AUTO);
+               CameraParameters::FOCUS_MODE_FIXED);
     params.set(CameraParameters::KEY_FOCUS_MODE,
-               CameraParameters::FOCUS_MODE_AUTO);
+               CameraParameters::FOCUS_MODE_FIXED);
 
     params.set(CameraParameters::KEY_FOCAL_LENGTH, "10.001");
     params.set(CameraParameters::KEY_HORIZONTAL_VIEW_ANGLE, "54.8");
-- 
1.8.0

